src/pgduckdb_hooks.cpp

```diff
@@ -353,6 +356,18 @@ DuckdbExplainOneQueryHook(Query *query, int cursorOptions, IntoClause *into, Exp
 	prev_explain_one_query_hook(query, cursorOptions, into, es, queryString, params, queryEnv);
 }
 
+static bool
+IsOutdatedMotherduckCatalogErrcode(int error_code) {
+	switch (error_code) {
+	case ERRCODE_UNDEFINED_COLUMN:
+	case ERRCODE_UNDEFINED_SCHEMA:
+	case ERRCODE_UNDEFINED_TABLE:
+		return true;
+	default:
+		return false;
+	}
```

Comment:
nit- let's use a non-branching version?
```suggestion
  return error_code == ERRCODE_UNDEFINED_COLUMN ||
         error_code == ERRCODE_UNDEFINED_SCHEMA ||
         error_code == ERRCODE_UNDEFINED_TABLE
```

Reply:
As discussed over chat, I'm keeping the old one because clang-format formats that better imo (possibly something we want to address separately). Since it's not in a hot path this seems totally fine.